controllers/contactController.js

```diff
@@ -0,0 +1,85 @@
+const asyncHandler = require("express-async-handler");
+const Contact = require("../models/contactModel");
+// desc Get all contacts
+// route GET/api/contacts
+// access private
+const getContacts = asyncHandler (async(req,res)=>{
+    const contacts=await Contact.find({user_id: req.user.id})
+    res.status(200).json(contacts);
+})
+
+// desc Create New contact
+// route POST/api/contacts
+// access private
+const createContact=asyncHandler(async(req,res)=>{
+    console.log("The request body is:",req.body);
+    const {name, email, phone}=req.body;
+    if(!name || !email || !phone){
+        res.status(400);
+        throw new Error("All fields are mandatory!")
+    }
+    const contact=await Contact.create({
+        name,
+        email,
+        phone,
+        user_id : req.user.id,
+    });
+    res.status(201).json(contact);
+})
+
+// desc Get contact
+// route GET/api/contacts/:id
+// access private
+const getContact=asyncHandler(async(req,res)=>{
+    const contact = await Contact.findById(req.params.id);
+    if(!contact){
+        res.status(404);
+        throw new Error("contact not found")
+    }
+    res.status(200).json(contact);
+})
+
+// desc update contact
+// route PUT/api/contacts/:id
+// access private
+const updateContact=asyncHandler(async(req,res)=>{
+    const contact = await Contact.findById(req.params.id);
+    if(!contact){
+        res.status(404);
+        throw new Error("contact not found")
+    }
+
+    if (contact.user_id.toString() !== req.user.id){
```

Comment:
Lets use single convention... since we are using camel case.... and not underscore